packages/react/package.json

```diff
@@ -39,7 +39,7 @@
     "dev": "tsup --watch",
     "dev:build": "tsup",
     "clean": "rimraf dist node_modules",
-    "lint": "tsc --noEmit --project tsconfig.dev.json && yarn eslint ./src --ext .js,.ts,.tsx",
+    "lint": "tsc --noEmit --project tsconfig.dev.json && eslint",
```

Comment:
Forgot to not that this change speeds up the lint command by abt `166%` (from approximately 16 seconds to 6 seconds, YMMV)